src/main/java/nextstep/subway/path/dto/PathResponse.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.subway.path.dto;
+
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PathResponse {
+
+    private List<StationResponse> stations;
+    private int distance;
+
+    public PathResponse() {
+
+    }
+
+    public PathResponse(List<StationResponse> stations, int distance) {
+        this.stations = stations;
+        this.distance = distance;
+    }
+
+    public static PathResponse of(List<Station> stationList, int distance) {
+        List<StationResponse> stationResponses = stationList.stream().map(StationResponse::of).collect(Collectors.toList());
```

Comment:
```suggestion
        List<StationResponse> stationResponses = stationList.stream()
        .map(StationResponse::of)
        .collect(Collectors.toList());
```
이렇게 개행해주셔도 좋을 것 같습니다!